src/components/SocialBar/SocialBar.st.css

```diff
@@ -1,10 +1,7 @@
-
-
 /*Defaults*/
 :vars {
   IconWidth: 24px;
   IconsGap: 40px;
-  /*Defaults params goes here*/
 }
 
 /*Overrides*/
```

Comment:
I think we can remove it if it's empty.

Reply:
Do you mean to remove `/*Defaults*/` comment when there are no default variables at all?

Reply:
I meant to remove this:

/*Overrides*/
:vars {	
  /*Deceleration of overridable placeholders*/
}